test/README.md

```diff
@@ -1,5 +1,10 @@
 
 # Conformance tests
+
+## Prerequisite(s)
+
+Install Ginkgo v2 CLI following the [Migration Guide](https://onsi.github.io/ginkgo/MIGRATING_TO_V2)
```

Comment:
Readme on how to configure testcases for soak test and run ?

Reply:
We already have a readme for test folder.
anyone who wants to run soak test are readers .

Reply:
I meant, is that doc meant for developers who would add tests, or run tests? For adding a new test, the doc contains more details.

Reply:
We are running serial and parallel tests together right now. Do we need to run the parallel tests (soak) separately and have a separate report?

Reply:
Yes , || tasks are  add on tests for specific cases.

Reply:
also when we move to v2 in cnf-feature deploy, we don't want to run any || cases . It should run basic validation tests only .
cnf-feature ptp tests are run on OCP nightly builds. So requirement is just make sure the infrastructure is available and ptp offsets are generated.